Source/Core/DolphinQt/Debugger/BreakpointWidget.cpp

```diff
@@ -188,7 +188,15 @@ void BreakpointWidget::Update()
     m_table->setItem(i, 3,
                      create_item(QStringLiteral("%1").arg(bp.address, 8, 16, QLatin1Char('0'))));
 
-    m_table->setItem(i, 4, create_item());
+    QString flags;
+
+    if (bp.break_on_hit)
+      flags.append(QLatin1Char{'p'});
```

Comment:
Why `p`? This should probably be replaced with `b` for `break` instead?

Reply:
I did this a while ago and don't remember why I chose `p`, so I'll change it to `b`.